bin/BuildSystem/BuildSystemBase.py

```diff
@@ -240,18 +240,17 @@ def internalPostInstall(self):
 
             regexp = re.compile('{.*} (.*)')
             for f in files:
-                expectedPdb = os.path.splitext(f)[0] + '.pdb'
-                if not os.path.exists(expectedPdb):
+                if not os.path.exists(os.path.splitext(f)[0] + '.pdb'):
```

Comment:
pleas use " quotes and format strings, either `f"{os.path.splitext(f)[0]}.pdb"` or `"{0}.pdb".format(os.path.splitext(f)[0])`

Reply:
Okay.